server/views/pages/temporaryabsences/confirmTemporaryAbsenceAddedToRoll.njk

```diff
@@ -1 +1,37 @@
-<h1>is on the establishment roll and is located in reception</h1>
\ No newline at end of file
+{% extends "../../partials/layout.njk" %}
+{% set pageTitle = "This person has returned to prison" %}
+
+{% block content %}
+    <div class="govuk-width-container">
+        <main class="govuk-main-wrapper">
+            <div class="govuk-grid-row">
+                <div class="govuk-grid-column-two-thirds">
+                    {{ govukPanel({
+                        titleText: firstName + " " + lastName + " " + "has returned to " + prison.description,
+                        html: "Prison number<br><strong>" + prisonNumber + "</strong>",
```

Comment:
is this prone to html injection? 

The prison number comes from user provided input?

Reply:
Think it's difficult (maybe impossible to exploit) due to the need to set up flash state before hitting the route.. 

Might be nice to see if there is another way to guarantee it is safe though

Reply:
DCS-1407 for this